README.md

```diff
@@ -9,6 +9,32 @@ rails db:migrate
 rails server
 ```
 
+#Dockerizing application
+
+To dockerize the treatment database application use the following commands after setting up docker on your local machine or server.
+
+```bash
+cd \\to_the_treatment_database_directory
+docker build -t treatment_database_image .
+docker run -it --rm treatment_database_image bundle exec rake test
+docker run -d --name treatment_database -itP -v $(pwd):/app treatment_database_image
```

Comment:
Add "-p 3000:3000" argument to bind ports so that you don't have to search for the host port

```bash
docker run -d --name treatment_database -p 3000:3000 -itP -v $(pwd):/app treatment_database_image
```